classes/cron.php

```diff
@@ -48,10 +48,11 @@ class cron {
      * @return bool
      */
     public static function run() {
-        echo "Starting coursework cron functions...\n";
+        if (!defined('PHPUNIT_TEST')) {
```

Comment:
If this needs to be avoided on test runs, there is also an equivalent for Behat

Reply:
In PHPUnit it triggers a "risky test" warning.  I think we'd be much less likely to run cron from behat, so less of an issue, but I changed this in commit bbba6a6 just in case